crates/polars-ops/src/frame/join/hash_join/single_keys_inner.rs

```diff
@@ -44,6 +44,8 @@ pub(super) fn hash_join_tuples_inner<T, I>(
     swapped: bool,
     validate: JoinValidation,
     join_nulls: bool,
+    // We should know the number of nulls to avoid extra calculation
+    build_null_count: usize,
```

Comment:
I wasn't sure about adding this param, but I wanted to avoid performing another pass on all values to calculate it inside the function.

Reply:
Yes, that's worth it.